admin/spec/back-end/helpers/pupil-status-presenter.spec.js

```diff
@@ -0,0 +1,86 @@
+/* global describe, expect, it */
+
+const moment = require('moment')
+
+const pupilStatusPresenter = require('../../../helpers/pupil-status-presenter')
+
+describe('pupilStatusPresenter', () => {
+  describe('getPresentationData', () => {
+    const checkWindowData = {
+      checkStartDate: moment.utc().subtract(3, 'days'),
+      checkEndDate: moment.utc().add(1, 'days')
+    }
+    it('collects pupils with errors', () => {
+      const pupils = [
+        {
+          status: 'Error in processing'
+        },
+        {
+          status: 'Incomplete'
+        }
```

Comment:
There should be at least one more status, or it *might* not be filtering at all.  This comment seems to apply to other tests here too.

Reply:
@jon-shipley added large data input for all included tests